packages/authentication/client/src/utils/init-ganache.js

```diff
@@ -0,0 +1,36 @@
+import {
+    ContractFactory,
+    Wallet,
+    providers,
+    utils,
+    BigNumber,
+  } from 'ethers';
+import ethrReg from '@ew-did-registry/did-ethr-resolver/src/constants/EthereumDIDRegistry.json';
+import { Methods } from '@ew-did-registry/did';  
+  const { hexlify } = utils;
+  
+  const GANACHE_PORT = 8544;
+  const provider = new providers.JsonRpcProvider(`http://localhost:${GANACHE_PORT}`);
+  
+  export const deployRegistry = async (fillAccounts) => {
+    const faucet = provider.getSigner(2);
+  
+    await Promise.all(fillAccounts.map((acc) => faucet.sendTransaction({
+      to: acc,
+      value: hexlify(BigNumber.from('1000000000000000000')),
+    })));
+  
+    const registryFactory = new ContractFactory(ethrReg.abi, ethrReg.bytecode,
+      new Wallet('0x49b2e2b48cfc25fda1d1cbdb2197b83902142c6da502dcf1871c628ea524f11b', provider));
+    const registry = await registryFactory.deploy();
+    // @TODO: deploy proxy factory
+    return registry.address;
+  };
+
+  export const getSettings = async (fillAccounts) => {
+    const registryAddress = await deployRegistry(fillAccounts)
+    const settings = { address: registryAddress, abi: ethrReg.abi, method: Methods.Erc1056};
+
+    return settings;
+  }
+  
```

Comment:
I saw that the function `getSettings`  were imported from `init-ganache` and used to set the resolver settings in client/src/utils/did. Since this file didn't exist I had an error when trying to run the client

Reply:
@JGiter @knzeng-e Are we removing the `authentication` package based on discussion from stand-up today? Or do you think it is still valuable?

Reply:
I think this may be removed from packages, since we have a iam-client-examples which helps illustrate this authentication process (if I understood correctly). What do you think @JGiter ?

Reply:
Right. We should separate examples from library